db/migrate/20200325164623_add_unique_indices.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class AddUniqueIndices < ActiveRecord::Migration[5.2]
```

Comment:
This I don't get--adding indices makes sense, but why did you have to specify that they are unique?

Reply:
This is a suggestion of rubocop-rails

https://github.com/rubocop-hq/rubocop-rails/blob/master/manual/cops_rails.md#railsuniquevalidationwithoutindex

Reply:
To be clear, they're not unique among indices, they're indices that ensure uniqueness.